third_party/move/move-model/src/model.rs

```diff
@@ -3131,7 +3133,7 @@ impl<'env> ModuleEnv<'env> {
             return deps;
         }
         for fun_env in self.get_functions() {
-            for used_fun in fun_env.get_used_functions().expect("used functions") {
+            for used_fun in fun_env.get_used_functions_with_transitive_inline() {
```

Comment:
Maybe add a brief comment here saying something about why we need to traverse within inline functions.